Lucidtech/Las/Client.cs

```diff
@@ -85,11 +85,19 @@ public object PostDocuments(string contentType, string consentId)
         /// An empty object 
         /// </returns>
         ///         
-        public object PutDocument(string documentPath, string contentType, string presignedUrl)
+        public object PutDocument(string documentPath, string contentType, string presignedUrl, 
+            Dictionary<string, string> kwargs = null)
```

Comment:
Is it better to rename "kwargs" to something more explainable like "additionalHeaders" or "headers" or "extraHeaders"?